CONTRIBUTING.md

```diff
@@ -34,6 +34,52 @@ Symfony projects for instance).
 * Check if tests pass: `vendor/bin/phpunit`.
 * Fix project itself: `php php-cs-fixer fix`.
 
+## Working With Docker
+
+This project provides a Docker setup that allows working on it using any of the supported PHP versions.
+
+To use it, you first need to install:
+
+ * [Docker](https://docs.docker.com/get-docker/)
+ * [Docker Compose](https://docs.docker.com/compose/install/)
+
+Make sure the versions installed support [Compose file format 3.8](https://docs.docker.com/compose/compose-file/).
+
+Next, copy [`docker-compose.override.yaml.dist`](./docker-compose.override.yaml.dist) to `docker-compose.override.yaml`
+and edit it to your needs. The relevant parameters that might require some tweaking have comments to help you.
+
+You can then build the images:
+
+```console
+$ docker-compose build --parallel
```

Comment:
```
ker@dus:~/github/PHP-CS-Fixer λ docker-compose build
ERROR: Version in "./docker-compose.yaml" is unsupported. You might be seeing this error because you're using the wrong Compose file version. Either specify a supported version (e.g "2.2" or "3.3") and place your service definitions under the `services` key, or omit the `version` key and place your service definitions at the root of the file to use version 1.
For more on the Compose file format versions, see https://docs.docker.com/compose/compose-file/
ker@dus:~/github/PHP-CS-Fixer λ docker-compose --version
docker-compose version 1.25.0, build unknown
ker@dus:~/github/PHP-CS-Fixer λ docker --version
Docker version 19.03.8, build afacb8b7f0
```

what am I doing wrong?

Reply:
Your Docker Compose version is too old to support version `3.8` of the Compose file format. But I don't think features specific to this version are used here so we can probably use a lower one.

Reply:
interesting, `sudo apt-get docker-compose` on ubuntu brings only 1.25.0 and does not want to offer anything newer.
(yes, i know, i can download the binaries manually...)